test/functional/cmdLineTests/criu/playlist.xml

```diff
@@ -85,6 +85,7 @@
 			<variation>-Xjit:count=0 -XX:+CRIURestoreNonPortableMode</variation>
 		</variations>
 		<command>
+			TR_Options=$(Q)exclude={org/openj9/criu/TimeChangeTest.nanoTimeInt()J},dontInline={org/openj9/criu/TimeChangeTest.nanoTimeInt()J|org/openj9/criu/TimeChangeTest.nanoTimeJit()J}$(Q) \
```

Comment:
I suggest move the JIT option into the variation section above, there is a mode `-Xint` apparently conflict with this `TR_Options`.

Reply:
`TR_Options` should not conflict with `-Xint` because the JIT will not be initialized and the env var will be ignored by the rest of the components (unless someone decides to use it for other purposes).

Initially I tried setting these in the variations section, but the braces need to be quoted or escaped when being passed through the shell and every combination I tried failed.

I think it's probably better to use the current approach anyhow because someone adding a variation in the future will need to be aware of whether or not the JIT and/or AOT compilers are enabled by their options and duplicate the string, otherwise the tests will be ineffective.

Reply:
@llxia any insights to add the command line option https://github.com/eclipse-openj9/openj9/pull/15609#discussion_r930019467 into `variation`?

Reply:
Yes, I think it is easier to put in command in this case.

Reply:
It appears there was a problem to do so - _the braces need to be quoted or escaped when being passed through the shell and every combination I tried failed._

Reply:
Had some offline discussion with Lan, the problem is between `cmdlinetester` and `criuScript.sh`, an additional option along w/ a separate playlist test target might be required to accommodate such JIT options hence out of scope of this PR.
There is no conflict settings introduced by `TR_Options` in this testcase, current approach is fine.